### PR TITLE
Ingest AMTA 1998

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Review by xinru1414 — APPROVED** [review]
LGTM

**Comment by mjpost** [comment]
@davidstap can you update this one, too? Should be mostly just the `<publisher>` tags, which might be Springer, or AMTA itself.

**Comment by davidstap** [comment]
Yes, I'll do that this weekend!

**Comment by davidstap** [comment]
should be good by now, if someone can approve I'll squash and merge.

**Comment by mjpost** [comment]
Looks fantastic. I spot checked a few URLs.

File: data/xml/1998.amta.xml [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+    </paper>
+    <paper id="8">
+      <title>An <fixed-case>E</fixed-case>nglish-to-<fixed-case>T</fixed-case>urkish interlingual <fixed-case>MT</fixed-case> system</title>
+      <author><first>Dilek Zeynap</first><last>Hakkani</last></author>
+      <author><first>Göklan</first><last>Tür</last></author>
+      <author><first>Kemal</first><last>Oflazer</last></author>
+      <author><first>Teruko</first><last>Mitamura</last></author>
+      <author><first>Eric H. Nyberg</first><last>3rd</last></author>
```
Comment (danielgildea) [inline data/xml/1998.amta.xml:144]: Nyberg, 3rd

File: data/xml/1998.amta.xml [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+      <title>Making semantic interpretation parser-independent</title>
+      <author><first>Ulrich</first><last>Germann</last></author>
+      <pages>286-299</pages>
+      <url>https://link.springer.com/chapter/10.1007/3-540-49478-2_26</url>
+      <abstract>We present an approach to semantic interpretation of syntactically parsed Japanese sentences that works largely parser-independent. The approach relies on a standardized parse tree format that restricts the number of syntactic configurations that the semantic interpretation rules have to anticipate. All parse trees are converted to this format prior to semantic interpretation. This setup allows us not only to apply the same set of semantic interpretation rules to output from different parsers, but also to independently develop parsers and semantic interpretation rules.</abstract>
+    </paper>
+    <paper id="26">
+      <title>Implementing <fixed-case>MT</fixed-case> in the <fixed-case>G</fixed-case>reek public sector</title>
+      <author><first>Athanassia</first><last>Fourla</last></author>
+      <author><first>lga</first><last>Yannoutsou</last></author>
```
Comment (danielgildea) [inline data/xml/1998.amta.xml:286]: Olga

File: data/xml/1998.amta.xml [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+      <abstract>We describe a streamlined knowledge acquisition method for semi-automatically constructing knowledge bases for a Knowledge Based Machine Translation (KBMT) system. This method forms the basis of a very simple Java-based user interface that enables a language expert to build lexical and syntactic transfer knowledge bases without extensive specialized training as an MT system builder. Following [Wu 1997], we assume that the permutation of binary-branching structures is a sufficient reordering mechanism for MT. Our syntactic knowledge is based on a novel, highly constrained grammar construction environment in which the only re-ordering mechanism is the permutation of binary-branching structures (Twisted Pair Grammar). We describe preliminary results for several fully implemented components of a Hindi/Urdu to English MT prototype being built with this interface.</abstract>
+    </paper>
+    <paper id="29">
+      <title>A thematic hierarchy for efficient generation from lexical-conceptual structure</title>
+      <author><first>Bonnie</first><last>Dorr</last></author>
+      <author><first></first><last>NizarHabash</last></author>
```
Comment (danielgildea) [inline data/xml/1998.amta.xml:309]: Nizar Habash

**Comment by mjpost** [comment]
Hi @davidstap, it looks like you missed @danielgildea's comments here pointing out a few typos. Can you include these with your next PR?

**Comment by davidstap** [comment]
whoops I thought they were fixes (commits) and not comments! I'll add them with the 2006 AMTA ingestion, apologies.

**Pull request merged**